tests/drivers/gpio/gpio_enable_disable_interrupt/dts/bindings/test-gpio-enable-disable-interrupt.yaml

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2022 Google Inc
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+    This binding provides resources required to build and run the
+    tests/drivers/gpio/gpio_enable_disable_interrupt test in Zephyr.
+
+compatible: "test-gpio-enable-disable-interrupt"
```

Comment:
I see two patterns for defining devicetree nodes used by tests.  Some tests define the schema using the `vnd,` prefix under [dts/bindings/tests](https://github.com/zephyrproject-rtos/zephyr/tree/main/dts/bindings/test) and a few tests create local dts/bindings folder like you did here.

It looks like it's more common use `vnd,` prefix.  Can you move this?

Reply:
Done